mail/mail-server/Portfile

```diff
@@ -68,6 +68,8 @@ variant logrotate \
     depends_lib-append  port:logrotate
 }
 
+default_variants-append +logrotate
```

Comment:
when changing the `default_variants` you should increase the `revision`

Reply:
Thanks done.